test/tokenize-arg-string.ts

```diff
@@ -1,58 +1,58 @@
 /* global describe, it */
+import { expect, should } from 'chai'
+import { tokenizeArgString } from '../lib/tokenize-arg-string'
 
-const tokenizeArgString = require('../lib/tokenize-arg-string')
+should()
```

Comment:
let's just switch to using `expect`, if it's not too much of a hassle, rather than doing back flips to address linting.

Reply:
> let's just switch to using `expect`, if it's not too much of a hassle, rather than doing back flips to address linting.

Done.

Reply:
I recommend keeping tests as they are written, some with should other with expect. Once again, we should not mix switching to ts and refactoring (especially refactoring tests).

Reply:
@mleguen are you ok with how I originally had the tests? This was mostly the same as they were written. I just added a couple new test cases and grouped the test cases based on the argument type.

Reply:
> I just added a couple new test cases

No problem with this

> and grouped the test cases based on the argument type

Maybe should we separate this refactoring?

Reply:
@mleguen updated as recommenced.

Reply:
Created a new issue to switch tests to use expect #275 to capture this feedback.